src/vma/dev/cq_mgr.cpp

```diff
@@ -660,6 +661,7 @@ bool cq_mgr::compensate_qp_poll_success(mem_buf_desc_t* buff_cur)
 		if (m_rx_pool.size() || request_more_buffers()) {
 			do {
 				mem_buf_desc_t *buff_new = m_rx_pool.get_and_pop_front();
+				buff_new->reset_ref_count();
```

Comment:
I think it is better to reset_red_count() inside cq_mgr::request_more_buffers()...